capstone/src/main/python/main.py

```diff
@@ -0,0 +1,123 @@
+from flask import Flask
+from google.cloud import datastore
+from sklearn.tree import DecisionTreeRegressor
+import numpy as np
+import pandas as pd
+app = Flask(__name__)
```

Comment:
Please pip install and run pylint on your python files:
https://google.github.io/styleguide/pyguide.html